Majasdarbs/03-majasdarbs/portfolio/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Nauris Kostjuks</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav class="navbar">
+      <ul>
+        <li><a href="#">Home</a></li>
+        <li><a href="#">About Me</a></li>
+        <li><a href="#">Portfolio</a></li>
+        <li><a href="#">Services</a></li>
+        <li><a href="#">Contact Me</a></li>
+      </ul>
+    </nav>
+    <section id="home">
+        <div class="main">
+            <h1 class="headings">I AM <br><span class="input"></span> </h1>
+        </div>
+        
+    </section>
+    <section id="about">
+        <h1 class="heading-about-me">ABOUT ME</h1>
+        <div id="pic">
+            <img src="about-me.jpg" alt="">
+            <div id="intro">
+                <h2>Nauris Kostjuks</h2>
+                <p class="about-me-text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Assumenda, Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ipsum reprehenderit assumenda ex consequuntur optio quibusdam nobis ratione necessitatibus quis. Quae debitis modi animi illo nobis natus id atque accusantium, placeat dolore harum blanditiis sunt esse laboriosam quasi ipsa autem! Aliquid qui voluptatibus molestiae numquam necessitatibus alias ducimus error quod itaque. Saepe corporis, praesentium et ullam ipsa ratione tempora aliquam dolorem optio aliquid at iste repellat dignissimos illo cupiditate dolore dolor accusamus porro sunt. Suscipit reiciendis possimus delectus nobis ullam debitis inventore quos, eius culpa aliquam magnam ex quia ipsa excepturi saepe rerum repellat, numquam perferendis? Fugit totam porro saepe quis!</iusto></p>
+            </div>
+        </div>
+
+    </section>
+    
+    <script type="text/javascript" src="js/script.js"></script>
```

Comment:
don't you import the same script below? not sure why do you need embed code locally, if you already use CDN
https://github.com/mattboldt/typed.js#cdn

and also script is located in the same directory, but you import it from js folder, which doesn't exist.

Remove this script tag and custom script.js file